lib/seed/user_seeder.rb

```diff
@@ -11,15 +13,36 @@ def initialize(config:)
       @logger = Rails.logger.child(class: self.class.name)
       @number_of_users_per_facility = config.max_number_of_users_per_facility
       @organization = Seed.seed_org
-      puts "Starting #{self.class} with #{config.type} configuration"
+      announce "Starting #{self.class} with #{config.type} configuration"
     end
 
     attr_reader :config
     attr_reader :logger
     attr_reader :number_of_users_per_facility
     attr_reader :organization
+    delegate :stdout, to: :config
 
     def call
+      create_dashboard_admins
+      create_mobile_users
+    end
+
+    private
+
+    def create_dashboard_admins
+      unless EmailAuthentication.exists?(email: "power_user@simple.org")
+        FactoryBot.create(:admin, :power_user, full_name: "Power User", email: "power_user@simple.org", password: config.admin_password)
+      end
+
+      unless EmailAuthentication.exists?(email: "cvho@simple.org")
+        fg_1, fg_2 = *FacilityGroup.take(2)
+        user = FactoryBot.create(:admin, :manager, full_name: "CVHO", email: "cvho@simple.org")
+        user.accesses.create! resource_id: fg_1.id, resource_type: "FacilityGroup"
+        user.accesses.create! resource_id: fg_2.id, resource_type: "FacilityGroup"
+      end
```

Comment:
Same here. Can we address the situation where a "cvho@simple.org" exists, but doesn't have the right permissions? A common pitfall would be - a "cvho@simple.org" exists, but has access records pointing to obsolete/deleted facilities.

Additionally, looks like we can use the `grant_access` method to _reset_ a user's permissions pretty efficiently.

```ruby
facility_groups = FacilityGroup.take(2)
facilities = facility_groups.flat_map(&:facilities)
cvho = User.find_by(email: "cvho@simple.org:") || FactoryBot.create(...)
power_user.grant_access(cvho, facilities)
```
What do you think?

Reply:
I added a bit more smarts to make sure the CVHO has access to two districts.

Looks like `grant_access` only knows about facility_ids, but we want to grant access at the facility_group level, so I went with a slightly different approach.